packages/tree-widget/src/components/ModelsTree.tsx

```diff
@@ -0,0 +1,157 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
```

Comment:
here too. mind taking a pass through the rest of the files?

Reply:
fixed. Also passed through the rest of the files ensuring that headers were correct